src/filesystem.cpp

```diff
@@ -18,6 +18,7 @@
 #ifdef _MSC_VER
 #   include "wdirent.h"
 #   include <direct.h>
+#   define chdir _chdir
```

Comment:
I must be missing something, why is this being adjusted for VS builds in an OSX PR?